### DISCUSSION TITLE
Modal Form/Tabs hide "save" Button

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#9762] [Help]
I have a Form inside a Modal. Its seperated in 3 Tabs.
Is it possible to hide/remove the "create/update" Button on specific Tabs? I would like only show it on Tab1.

**Comment by leandrocfe** [comment]
Could you share the code please?

**Comment by photojo** [comment]
Did you find a solution? Can you share?
Is there no way to have several forms in one modal?

**Comment by otavio-araujo** [comment]
Did anyone get the solution?

**Comment by e2d2-dev** [comment]
Just create an action in each tab?!?

`Actions::create{[
    Action::make('saveTab1')->action()
]}`

**Reply by otavio-araujo** [reply]
Yea, make sense!

Thank you very much!
